src/Hl7.Fhir.Core.Tests/TestData/summary/bundle-summary-text.json

```diff
@@ -1 +1 @@
-{"resourceType":"Bundle","id":"my-bundle","type":"searchset","total":1803,"entry":[{"fullUrl":"http://base/Patient/patient-one","resource":{"resourceType":"Patient","id":"patient-one","meta":{"versionId":"eric-clapton","tag":[{"system":"http://hl7.org/fhir/v3/ObservationValue","code":"SUBSETTED"}]},"text":{"div":"<div xmlns='http://www.w3.org/1999/xhtml'>A great blues player</div>"}},"search":{"mode":"match"}},{"fullUrl":"http://base/Patient/patient-two","resource":{"resourceType":"Patient","id":"patient-two","meta":{"versionId":"bb-king","tag":[{"system":"http://hl7.org/fhir/v3/ObservationValue","code":"SUBSETTED"}]},"text":{"div":"<div xmlns='http://www.w3.org/1999/xhtml'>Another great blues player</div>"}},"search":{"mode":"match"}}]}
\ No newline at end of file
+{"resourceType":"Bundle","id":"my-bundle","type":"searchset","total":1803,"entry":[{"fullUrl":"http://base/Patient/patient-one","resource":{"resourceType":"Patient","id":"patient-one","meta":{"versionId":"eric-clapton","tag":[{"system":"http://hl7.org/fhir/v3/ObservationValue","code":"SUBSETTED"}]},"text":{"div":"<div xmlns='http://www.w3.org/1999/xhtml'>A great blues player</div>"},"active":true,"name":[{"use":"official","family":"Clapton"}],"gender":"male","birthDate":"2015-07-09"},"search":{"mode":"match"}},{"fullUrl":"http://base/Patient/patient-two","resource":{"resourceType":"Patient","id":"patient-two","meta":{"versionId":"bb-king","tag":[{"system":"http://hl7.org/fhir/v3/ObservationValue","code":"SUBSETTED"}]},"text":{"div":"<div xmlns='http://www.w3.org/1999/xhtml'>Another great blues player</div>"},"active":true,"name":[{"use":"nickname","family":"King"}]},"search":{"mode":"match"}}]}
```

Comment:
I feel there should not be any change to how summaries are done.  Especially because it seems elements like "active" and "name" are now included - but they are not mandatory?

Reply:
This is not correct - I have pushed an update that fixes this.